packages/typespec-client-generator-core/src/interfaces.ts

```diff
@@ -599,3 +601,105 @@ export enum UsageFlags {
   // Input will also be set when MultipartFormData is set.
   MultipartFormData = 1 << 5,
 }
+
+interface SdkExampleBase {
+  kind: string;
+  name: string;
```

Comment:
So, all ##Example is indexed by this "name=title"?

Reply:
yes. the name is the title of the example.